docs/shared/packages/nuxt/nuxt-plugin.md

```diff
@@ -50,3 +50,87 @@ pnpm install -D @nx/nuxt
 ```shell
 nx g @nx/nuxt:app my-app
 ```
+
+## Add Nx to existing Nuxt applications
+
+Using the `@nx/nuxt/plugin`, you can adds Nuxt targets to Nx. This allows you to run Nuxt for projects that have a `nuxt.config.ts` file via Nx with caching and everything setup.
```

Comment:
I think we need a doc to explain the plugin system and point to it here. Something @isaacplmann can help with?

The mental model is adding a plugin allows the commands to run through Nx, with caching, distribution, etc.